carbon-design-system/carbon-for-ibm-dotcom
# fix(card-pictogram): show body copy on both pictogram positions

### Related Ticket(s)

https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues/8302

### Description

- This PR fixes the issue where animation is not being applied to Card Pictorgram with the top position. This was an issue seen within the Web Component (React Wrapper).


### Changelog

**Changed**

- Set the body copy knob to show for both `top` and `bottom` position and not just `bottom`

### Testing

- Visit the Web Components (React Wrapper) storybook preview
- Navigate to `carbon-for-ibm-dotcom/canary/web-components-react/?path=/story/components-card--pictogram`
- Ensure that the body copy and animation get render correct for the "top" and " bottom" pictogram positions.
